csrc/python_frontend/translation_utils.h

```diff
@@ -0,0 +1,116 @@
+// clang-format off
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2023-present NVIDIA CORPORATION & AFFILIATES.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+// clang-format on
+#pragma once
+#include <ir/all_nodes.h>
+#include <ops/all_ops.h>
+
+namespace nvfuser::python_frontend {
+
+// Get std::function for BinaryOp
+template <typename ResultType, typename... ArgTypes>
+std::function<ResultType(ArgTypes...)> getFunction(const BinaryOp* bop) {
+  auto get_std_function = [](ResultType (*fn)(ArgTypes...)) {
+    return static_cast<ResultType (*)(ArgTypes...)>(fn);
```

Comment:
Do we need to have a static_cast here? I see that we are using `get_std_function` for overload resolution, but I don't see why we need a cast to the same function pointer?

Reply:
another nitpick, `get_std_function` sounds like we are getting std::..., but this is just our ops/arith.h stuff right?

Reply:
I was seeing an error before but it works now.
I renamed `get_std_function` to `wrap_function`.